CMakeLists.txt

```diff
@@ -76,9 +76,9 @@ include(cmake/Format.cmake)
 include(cmake/Lint.cmake)
 
 # regression test reference data
-set(REGRESSION_GOLD_STANDARD_VER 25 CACHE STRING "Version of gold standard to download and use")
+set(REGRESSION_GOLD_STANDARD_VER 26 CACHE STRING "Version of gold standard to download and use")
 set(REGRESSION_GOLD_STANDARD_HASH
-  "SHA512=314dc8312366d81ba33d1fde25812e9a7697b2f529de29e22662df0d458f1c4bc5b5bb4e649888170f66ffec0df1be20a9cf401944531c1c1ad835e26eaad28f"
+  "SHA512=c3f3a2da823a7e65a5800c902d26b3891f0a129812031ceae46e5c394f9c5bea36e26f645007422dca46dfdc7c446271c1ff5c0cec851bc40a186aa874beb643"
```

Comment:
I see the gold files have been updated, but I'm not sure I see the code that now requires the new files to pass.

Reply:
Oops you're right... A boundary flux should now be output in the hdf5 output test. I must have forgotten to commit that. Will fix it.

Reply:
Should be resolved now.